content/projects.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: base
+title: Project Proposals
+description: list of projects
+permalink: projects.html
+---
+
+# {{page.title}}
+
+Below is a list of internship projects which are continually being updated and added to. We also welcome project proposals from both prospective students and interested organisations.  Please contact us at our [mailbox](england.tdau@nhs.net) if you would have queries or would like to discus a proposal.
```

Comment:
Is this the right mailbox for now?

Reply:
`discus` -> `discuss`

Reply:
find and replace to datascience@nhs.net

Reply:
Three corrections to <a href="mailto:datascience@nhs.net">our mailbox</a>